evals/scripts/instructions.json

```diff
@@ -3,5 +3,6 @@
   "additional_instruction": "1. Recommendations (2-3 Sentences): Provide actionable strategies to improve engagement.\n\n2. Trends and Insights (2-3 Sentences): Identify emerging trends or insights from the data and compare to broader industry trends if relevant.",
   "get_campaign_score": "The guidelines below outline responses based on question type:\n\n\"\n- Fan & Score Queries: Provide answer with fan name & score points.",
   "get_tiktok_video_comments": "**Count Queries:** - Only provide the number when specifically asked for a numeric count of comments",
-  "get_tiktok_analysis": "Analyze the provided TikTok comment data to identify distinct fan segments. Leverage advanced natural language processing to generate insightful and sophisticated segment names that go beyond literal interpretations of the comments and capture the underlying motivations, sentiments, and engagement patterns of the user groups. The segment names should be concise yet evocative, employing professional and descriptive language. Present the results as a list of segment names and their corresponding user counts, formatted as: Segment Name: Count. Precede this with a brief artist profile: name, location, fan count, and follower count. Prioritize generating unique and insightful segment names, even if some segments have a small number of users."
+  "get_tiktok_analysis": "Analyze the provided TikTok comment data to identify distinct fan segments. Leverage advanced natural language processing to generate insightful and sophisticated segment names that go beyond literal interpretations of the comments and capture the underlying motivations, sentiments, and engagement patterns of the user groups. The segment names should be concise yet evocative, employing professional and descriptive language. Present the results as a list of segment names and their corresponding user counts, formatted as: Segment Name: Count. Precede this with a brief artist profile: name, location, fan count, and follower count. Prioritize generating unique and insightful segment names, even if some segments have a small number of users.",
+  "get_fan_segments": "Analyze the provided TikTok comment data to identify distinct fan segments. Leverage advanced natural language processing to generate insightful and sophisticated segment names that go beyond literal interpretations of the comments and capture the underlying motivations, sentiments, and engagement patterns of the user groups. The segment names should be concise yet evocative, employing professional and descriptive language. Present the results as a list of segment names and their corresponding user counts, formatted as: Segment Name: Count. Prioritize generating unique and insightful segment names, even if some segments have a small number of users."
```

Comment:
Do we have an `eval` for `get_fan_segments` instructions?
- expected: `evals` always written `before` new instructions.